distribution/docs/src/main/jdocs/content/_security-framework/realms-contents.adoc

```diff
@@ -0,0 +1,107 @@
+:title: Realms
+:type: subSecurityFramework
+:status: published
+:parent: Subject
+:order: 01
+:summary: Shiro realms.
+
+${branding} uses Realms for Authentication and Authorization.
+
+===== Authenticating Realms
+
+`org.apache.shiro.realm.AuthenticatingRealm`
+
+Authenticating Realms are used to authenticate an incoming authentication token and create a Subject on successful authentication.
+
+.Implementations of Authenticating Realms in ${branding}:
+[cols="1m,2" options="header"]
+|===
+
+|Classname
+|Description
+
+|${ddf-branding-lowercase}.security.realm.sts.StsRealm
+|This realm delegates authentication to the Secure Token Service (STS). It creates a `RequestSecurityToken` message from the incoming Authentication Token and converts a successful STS response into a Subject.
+
+|===
+
+===== Authorizing Realms
+
+`org.apache.shiro.realm.AuthorizingRealm`
+
+Authorizing Realms are used to perform authorization on the current Subject.
+These are used when performing both Service AuthZ and Filtering.
+They are passed in the `AuthorizationInfo` of the Subject along with the Permissions of the object wanting to be accessed. The response from these realms is a true (if the Subject has permission to access) or false (if the Subject does not).
+
+.Other implementations of the Security API within ${branding}
+[cols="1m,2" options="header"]
+|===
+
+|Classname
+|Description
+
+|org.codice.${ddf-branding-lowercase}.platform.filter.delegate.DelegateServletFilter
+|The `DelegateServletFilter` detects any servlet filters that have been exposed as OSGi services implementing `org.codice.ddf.platform.filter.SecurityFilter` and places them in-order in front of any servlet or web application running on the container.
+
+|org.codice.${ddf-branding-lowercase}.security.filter.websso.WebSSOFilter
+|This filter serves as the main security filter that works in conjunction with a number of handlers to protect a variety of contexts, each using different authentication schemes and policies.
```

Comment:
❓  What is a `context` in this context?

Reply:
I changed it to `web contexts`. I hope that is clearer and accurate.